src/gatling/scala/uk/gov/hmcts/reform/sendletter/actions/LettersService.scala

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.sendletter.actions
+
+import io.gatling.core.Predef._
+import io.gatling.core.structure.ChainBuilder
+import io.gatling.http.Predef._
+import io.gatling.http.HeaderValues._
+import io.gatling.http.HeaderNames._
+
+object LettersService {
+
+  val create: ChainBuilder =
+    exec(
+      http("Create letter")
+        .post("/letters")
+        .headers(Map(
+          "ServiceAuthorization" -> "Bearer ${service_token}",
+          ContentType -> ApplicationJson
+        ))
+        .body(StringBody(
```

Comment:
It's quite important for the body to be similar to the real one. Ideally, should be read from a file.

Reply:
Oh, and we have to provide some random values so that the producer doesn't treat those documents as one.